packages/web-app-files/src/views/spaces/GenericSpace.vue

```diff
@@ -518,6 +522,17 @@ export default defineComponent({
 
     const createNewFolderAction = computed(() => unref(createNewFolder)[0].handler)
 
+    const getIndicators = (resource: Resource) => {
+      if (unref(folderView).name !== FolderViewModeConstants.name.tiles) {
+        return resource.indicators
+      }
+
+      const visibleTilesViewIndicators = ['resource-locked', 'resource-processing']
```

Comment:
What do you think about introducing a new variable on the indicator interface, named like `critical`, `system`, whatever, and filter for that? I.e. tiles view only shows indicators that are `critical` while other views can decide to show all indicators? Don't know if this is a good solution, but I don't like to filter on the names of the indicators.

Reply:
(this would need to be moved to the `ResourceTiles` component as well if you agree with my other comment on this file).

Reply:
> What do you think about introducing a new variable on the indicator interface, named like `critical`, `system`, whatever, and filter for that? I.e. tiles view only shows indicators that are `critical` while other views can decide to show all indicators? Don't know if this is a good solution, but I don't like to filter on the names of the indicators.

Suggesting variable name "category" and values could be "sharing" and "system"